lib/widgets/totals_table.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import '../models/cow_requirements_model.dart';
+
+class TotalsTable extends StatelessWidget {
+  final List<Map<String, dynamic>> fodderItems;
+  final List<Map<String, dynamic>> concentrateItems;
+  final CowRequirements cowRequirements;
+
+  TotalsTable({
+    super.key,
+    required this.fodderItems,
+    required this.concentrateItems,
+    required this.cowRequirements,
+  });
+
+  final List<Map<String, dynamic>> columns = const [
+    {
+      'label': 'DM Intake\n(kg/d)',
+      'key': 'dmIntake',
+      'decimals': 5,
```

Comment:
Let's change this so that there are 2 decimals. Just like the cow requirements at the top.